pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from collections import Counter
+from optparse import make_option
+from django.contrib.auth import get_user_model
+from pootle_app.management.commands import PootleCommand
+from pootle_store.models import Unit
+
+
+User = get_user_model()
+
+
+class Command(PootleCommand):
+    option_list = PootleCommand.option_list + (
+        make_option("--from-revision", type=int, default=0, dest="revision",
+                    help="Only count contributions newer than this revision"),
+    )
+
+    help = "Print a list of contributors."
+
+    def handle_all(self, **options):
+        units = Unit.objects.values("submitted_by").exclude(submitted_by=None)
```

Comment:
I assume using `Unit` instead of `ScoreLog` is because we can get the list of contributors for currently active translations, and not get some history of contributors including those whose contributions have been dropped. Am I right?